amazon_msk/README.md

```diff
@@ -46,6 +55,12 @@ See [service_checks.json][14] for a list of service checks provided by this inte
 
 Need help? Contact [Datadog support][15].
 
+## Further Reading
+
+Additional helpful documentation, links, and articles:
+
+- [Monitor Amazon Managed Streaming for Apache Kafka with Datadog][16]
```

Comment:
Should this be sentence case?

Reply:
I often times go with the title case that's in the blog title 😅